src/Umbraco.Web.UI.Client/src/less/property-editors.less

```diff
@@ -903,9 +903,27 @@ ul.color-picker li a  {
 // Tags
 // --------------------------------------------------
 .umb-tags{border: @gray-10 solid 1px; padding: 10px; font-size: 13px; text-shadow: none;}
-.umb-tags .tag{cursor: pointer;  margin: 7px; padding: 7px; background: @turquoise}
-.umb-tags .tag i{padding: 2px;}
-.umb-tags input{border: none; background: @white}
+.umb-tags .tag{
+    cursor: default;
+    margin: 7px;
+    padding: 10px;
+    background: @turquoise;
+    position: relative;
+
+    .umb_confirm-action__overlay.-left{
+        top: 6px;
+        left: auto;
+        right: 15px;
+    }
+}
+.umb-tags .tag .icon-trash{
+    cursor: pointer;
+    padding: 2px;
+    font-size: 12px;
+    position: relative;
+    right: -6px;
+}
+.umb-tags input{border: none; background: @white;}
```

Comment:
Maybe restructure this to the following? :)

```
.umb-tags {
    border: @gray-10 solid 1px;
    padding: 10px;
    font-size: 13px;
    text-shadow: none;

    .tag {
        cursor: default;
        padding: 10px;
        background: @turquoise;
        position: relative;

        .icon-trash {
            cursor: pointer;
            padding: 2px;
            font-size: 12px;
            position: relative;
            right: -6px;
        }

        .umb_confirm-action__overlay.-left {
            top: 6px;
            left: auto;
            right: 15px;
        }
    }
    
    input {
        border: none;
        background: @white;
    }
}
```

Reply:
I can do that - No problem.

Reply:
Done :)